static/templates/user_profile_modal.hbs

```diff
@@ -6,7 +6,7 @@
                     <span class="{{user_circle_class}} user_circle user_profile_presence"></span>
                 </div>
                 <h1 class="modal__title" id="name">
-                    {{full_name}}
+                    <span>{{full_name}}</span>
```

Comment:
These CSS selectors are way too general. Can you rename `#name` to `.user_profile_name_heading` and add a `user_profile_name` CSS class on this spam, and then have the CSS rules apply to those?

Reply:
> These CSS selectors are way too general. Can you rename `#name` to `.user_profile_name_heading` and add a `user_profile_name` CSS class on this spam, and then have the CSS rules apply to those?

Applied these changes.